docs/install/prerequisites.md

```diff
@@ -1,21 +1,13 @@
 # Prerequisites
 
-__Portable (CO:RE) option:__
-* Linux kernel version >= 4.18
-* BTF enabled (_You can manually detect if your environments supports it by checking if the following file exists on your machine: /sys/kernel/btf/vmlinux or consult the following documentation: https://github.com/libbpf/libbpf#bpf-co-re-compile-once--run-everywhere)_
+* Linux kernel version >= 4.18.
+* libc, and the libraries: libelf, zlib.
```

Comment:
This is not a requirement for CO-RE option (as the object is already compiled).
Also, it is not a requirement when running from the docker image

Reply:
i thought libelf and zlib are runtime requirements of libbpf, no?

Reply:
We are using libbpf.a which statically compiles these libs into it